cmd/notification.go

```diff
@@ -290,12 +288,12 @@ func (sys *NotificationSys) DownloadProfilingData(ctx context.Context, writer io
 // SignalService - calls signal service RPC call on all peers.
 func (sys *NotificationSys) SignalService(sig serviceSignal) []NotificationPeerErr {
 	var idx = 0
-	ng := WithNPeers(len(sys.peerRPCClientMap))
-	for addr, client := range sys.peerRPCClientMap {
-		client := client
+	ng := WithNPeers(len(sys.peerHosts))
+	for _, addr := range sys.peerHosts {
```

Comment:
We don't need idx anymore it was used with map previously, since now we have peerHosts as slice we can simply do 

```
for idx, addr := range sys.peerHosts {
```